src/mol-util/param-definition.ts

```diff
@@ -548,6 +550,91 @@ export namespace ParamDefinition {
         }
     }
 
+    function normalizeParam(p: Any, value: any, defaultIfUndefined: boolean, prune: boolean): any {
```

Comment:
with prune = true, the group params return object will only contain current properties, without it, all old properties and there values are copied over

what is the use-case for prune = false? why keep the old properties around? for testing?

Reply:
Because of things like this https://github.com/molstar/molstar/blob/master/src/mol-theme/color/operator-name.ts#L46 (not pointing finders, but ... :D ) 

I wonder if this part deserves refactoring and how would we go around with that.

Reply:
> Because of things like this https://github.com/molstar/molstar/blob/master/src/mol-theme/color/operator-name.ts#L46

I think we can move those into the getParams function.

>  (not pointing finders, but ... :D )

guilty as charged

Reply:
moved the label options out, see 8d6557e51cf8367fc01d30ebf7ebe8bb5cc9ac2f